CHANGELOG.md

```diff
@@ -4,6 +4,49 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+
+## 2.17.0 (2024-09-20)
+
+This release contains performance improvements and bug fixes since
+the 2.16.1 release. We recommend that you upgrade at the next
+available opportunity.
+
+In addition, it includes these noteworthy features:
+
+* Full PostgreSQL 17 support for all existing features
+
+
+**PostgreSQL 14 deprecation announcement**
+We will continue supporting PostgreSQL 14 until April 2025. Sooner to that time, we will announce the specific version of TimescaleDB in which PostgreSQL 14 support will not be included going forward.
+
+**Features**
+* #6882: Allow delete on the compressed chunks without decompression.
```

Comment:
This sounds really deceptive - let's rephrase a little bit

```suggestion
* #6882: Allow delete of fully covered compressed batches on compressed chunks without decompression.
```

@svenklemm agree?

Reply:
Edited, please check.

Reply:
Hmm not sure your wording makes it more clear what this is about. This is about deleting full segments without decompression.

Reply:
Edited, please check